database.tf

```
@@ -3,10 +3,12 @@ locals {
 }
 
 resource "postgresql_role" "owner" {
-  name     = local.owner
-  login    = var.owner_password != null ? true : false
-  password = var.owner_password
-  roles    = var.roles
+  name            = local.owner
+  login           = var.owner_password != null ? true : false
+  create_database = var.owner_create_database != null ? true : false
+  create_role     = var.owner_create_role != null ? true : false
```

We used the ternary operator for login as it's computed with a string field, but for these 2 I think we can default them to false and set the value directly

```suggestion
  create_database = var.owner_create_database
  create_role     = var.owner_create_role
```